src/TraceEvent/Microsoft.Diagnostics.Tracing.TraceEvent.props

```diff
@@ -42,8 +42,15 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
       <Visible>False</Visible>
     </None>
-    <None Condition="Exists('$(MSBuildThisFileDirectory)..\lib\netstandard1.6\OSExtensions.dll')" Include="$(MSBuildThisFileDirectory)..\lib\netstandard1.6\OSExtensions.dll">
-      <Link>OSExtensions.dll</Link>
+
+	<!-- you have to pick the right version of this DLL because it depends on things besides System.Runtime.dll -->
+    <None Condition="'$(TargetFrameworkIdentifier)' != '.NETFramework' AND Exists('$(MSBuildThisFileDirectory)..\lib\netstandard1.6\OSExtensions.dll')" Include="$(MSBuildThisFileDirectory)..\lib\netstandard1.6\OSExtensions.dll">
+      <Link>OsExtensions.dll</Link>
```

Comment:
@vancem this breaks the fix from https://github.com/Microsoft/perfview/pull/781 where link to O**s**Extensions has been changed to O**S**Extensions (uppercase S) to work properly on Unix